code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -163,6 +163,12 @@
 			download_objective.owner = owner
 			download_objective.gen_amount_goal()
 			add_objective(download_objective)
+
+		else if(prob(15) && !(locate(/datum/objective/bankrupt) in objectives) && !(owner.assigned_role in list("Captain", "Head of Personnel", "Quartermaster")))
```

Comment:
Probably every head should be excluded from this

Reply:
Would be pretty funny to have heads trying to bankrupt other departments